packages/ui/react-ui-theme/src/styles/fragments/surface.ts

```diff
@@ -11,8 +11,13 @@
 // NOTE: This should align with theme's root --surface-bg.
 export const baseSurface = 'bg-neutral-12 dark:bg-neutral-900';
 
+export const baseSurfaceBorder = 'border-neutral-12 dark:border-neutral-900';
```

Comment:
I'd meant to solve this problem (or a related problem at least) using CSS variables so `var(--surface-bg)` is reliably the value of the background color of the current surface. It just requires a bit of wrangling in `theme.css`.